openpype/pipeline/farm/pyblish_functions.py

```diff
@@ -211,9 +211,15 @@ def create_skeleton_instance(
         log.warning(("Could not find root path for remapping \"{}\". "
                      "This may cause issues.").format(source))
 
-    family = ("render"
-              if "prerender.farm" not in instance.data["families"]
-              else "prerender")
+    # pass families from original instance to skeleton
+    family = "render"
+    # try to find family with farm suffixes
+    families = instance.data["families"]
+    for family_ in families:
+        match = re.search(r"\.farm(_frames)?$", family_)
+        if match:
+            family = re.sub(r"\.farm(_frames)?$", "", family_)
+            break
```

Comment:
```suggestion
    # search for family with farm suffixes to
    # pass families from original instance to skeleton
    families = instance.data["families"]
    farm_suffixes = [".farm", ".farm_frames"]
    for family_ in families:
        suffix = next((s for s in farm_suffixes if family.endswith(suffix)), None)
        if suffix:
            family = family_[:-len(suffix)]
            break
    else:
        # No family found with farm suffix
        family = "render"
```
This would likely be faster (and maybe more readable?)

Reply:
> I had to remove this hard-code in my fork of OP because we are publishing plates, reference and other families in the farm

@fabiaserra proposed code is suggesting that since you set target to farm suffixe `.farm` or `.farm_frames` will be added to what ever base family [product type]. The `render` should be onlly fallback for those missing the suffix - mostly the backward compatibility.